src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Navbar.css'
+
+export const Navbar = () => {
+  const getClassName = ({ isActive }: { isActive: boolean }): string => {
+    return isActive ? 'active-link' : ''
+  }
+  return (
+    <nav>
+      <a className={`link ${getClassName}`} href="/">
```

Comment:
`getClassName` is a function but we are not calling it, Is it working as expected?

Reply:
Fixed it with different approach.